sdk/spring/azure-spring-boot-test-aad/src/test/java/com/azure/test/aad/selenium/oauth2client/issuedat/AADOauth2AuthorizedClientCachedIT.java

```diff
@@ -31,12 +33,11 @@ public void testOauth2AuthorizedClientCached() {
         properties.put(
             "azure.activedirectory.authorization-clients.office.scopes",
             "https://manage.office.com/ActivityFeed.Read, "
-                + "https://manage.office.com/ActivityFeed.ReadDlp, "
```

Comment:
We should delete all contents about office scopes. And add "arm" related contents.